gitless/cli/gl_branch.py

```diff
@@ -203,3 +210,23 @@ def _do_set_head(commit_id, repo):
   pprint.ok(
       'Head of current branch {0} is now {1}'.format(curr_b, pprint.commit_str(commit)))
   return True
+
+
+def _do_rename(rename_b, repo):
+  ret = True
+  if len(rename_b) == 1 :
+    # Renaming this current branch
+    curr_b = repo.current_branch
+    curr_b.rename(rename_b[0])
+    pprint.ok('Renamed this branch to {0}'.format(rename_b[0]))
+  elif len(rename_b) == 2:
+    # Renaming a specified branch to a new name
+    b = helpers.get_branch(rename_b[0], repo)
+    b.rename(rename_b[1])
+    pprint.ok('Renamed branch {0} to {1}'.format(rename_b[0], rename_b[1]))
+  else :
+    # Gave more than 2 arguments
+    pprint.err(
+        'Too many arguments given')
```

Comment:
Improve the error msg a bit? Perhaps you can say that you expect 1 or 2 arguments?